src/components/Shared/Navbar/index.scss

```diff
@@ -0,0 +1,21 @@
+.navbar {
+  margin: auto;
+  max-width: 78rem;
+
+  &__container {
+    align-items: center;
+    display: flex;
+    height: 4.5rem;
+    justify-content: space-between;
+
+    &__logo {
+      align-items: center;
+      display: flex;
```

Comment:
consider defining a helper class to reduce duplicate [style](https://github.com/ahr-wtag/react-todo-app/pull/1/files#diff-dc660cd2ea7a53e722b25c363946b91a3e2bd1ea1095f9c2bb7a8182a52a5b8eR7).